src/test/java/com/puppycrawl/tools/checkstyle/utils/ScopeUtilsTest.java

```diff
@@ -149,4 +172,124 @@ public void testIsLocalVariableDefUnexpected() {
+        final DetailAST ast = new DetailAST();
+        ast.setType(TokenTypes.LITERAL_NEW);
+        final DetailAST ast2 = new DetailAST();
+        ast2.setType(TokenTypes.IDENT);
+        ast.addChild(ast2);
+        Assert.assertEquals(Scope.ANONINNER, ScopeUtils.getSurroundingScope(ast2));
+    }
+
+    @Test
+    public void testIsInInterfaceBlock() {
+        final DetailAST ast = new DetailAST();
+        ast.setType(TokenTypes.INTERFACE_DEF);
+        final DetailAST ast2 = new DetailAST();
+        ast2.setType(TokenTypes.OBJBLOCK);
+        ast.addChild(ast2);
+        final DetailAST ast3 = new DetailAST();
+        ast3.setType(TokenTypes.CLASS_DEF);
+        ast2.addChild(ast3);
+        final DetailAST ast4 = new DetailAST();
+        ast4.setType(TokenTypes.MODIFIERS);
+        ast3.addChild(ast4);
+        Assert.assertTrue(ScopeUtils.isInInterfaceBlock(ast3));
+        Assert.assertFalse(ScopeUtils.isInInterfaceBlock(ast4));
+    }
+
+    @Test
+    public void testIsInAnnotationBlock() {
+        final DetailAST ast = new DetailAST();
+        ast.setType(TokenTypes.ANNOTATION_DEF);
+        final DetailAST ast2 = new DetailAST();
+        ast2.setType(TokenTypes.OBJBLOCK);
+        ast.addChild(ast2);
+        final DetailAST ast3 = new DetailAST();
+        ast3.setType(TokenTypes.INTERFACE_DEF);
+        ast2.addChild(ast3);
+        final DetailAST ast4 = new DetailAST();
+        ast4.setType(TokenTypes.MODIFIERS);
+        ast3.addChild(ast4);
+        Assert.assertTrue(ScopeUtils.isInAnnotationBlock(ast3));
+        Assert.assertFalse(ScopeUtils.isInAnnotationBlock(ast4));
+    }
+
+    @Test
+    public void testisInInterfaceOrAnnotationBlock() {
+        Assert.assertTrue(ScopeUtils.isInInterfaceOrAnnotationBlock(
+                getNode(TokenTypes.ANNOTATION_DEF, TokenTypes.OBJBLOCK)));
+        Assert.assertTrue(ScopeUtils.isInInterfaceOrAnnotationBlock(
+                getNode(TokenTypes.INTERFACE_DEF, TokenTypes.OBJBLOCK)));
+        Assert.assertFalse(ScopeUtils.isInInterfaceOrAnnotationBlock(
+                getNode(TokenTypes.CLASS_DEF, TokenTypes.OBJBLOCK)));
+        Assert.assertFalse(ScopeUtils.isInInterfaceOrAnnotationBlock(
+                getNode(TokenTypes.LITERAL_NEW, TokenTypes.IDENT)));
+        Assert.assertFalse(ScopeUtils.isInInterfaceOrAnnotationBlock(
+                getNode(TokenTypes.ENUM_DEF, TokenTypes.OBJBLOCK)));
+    }
+
+    private static DetailAST getNode(int parentType, int nodeType) {
+        final DetailAST ast = new DetailAST();
+        ast.setType(parentType);
+        final DetailAST ast2 = new DetailAST();
+        ast2.setType(nodeType);
+        ast.addChild(ast2);
+        return ast2;
+    }
+
+    private static DetailAST getNodeWithParentScope(int literal, String scope,
+                                                    int parentTokenType) {
+        final DetailAST ast1 = new DetailAST();
```

Comment:
In every other test, you name variables `ast`, `ast2`, etc. and you put them in order of variable names. This is the only one that breaks that pattern.
Can we make the layout here similar?